common.sh

```diff
@@ -23,6 +23,15 @@ if [ "${SUPPORTS_AP}" != "yes" ]; then
 	read -n 1 -s -r -p "Press any key to continue, or CTRL+C to exit"
 fi
 
+function run_helper_script () {
+    if [ -f "scripts/${1}.sh" ]; then
+        echo "Running helper script '${1}'"
+        source "scripts/${1}.sh"
+    else
+        echo "Helper script '${1}' not found, skipping"
```

Comment:
This maybe unnecessary output, but useful for users to see what helper scripts they could included.

Reply:
I think this would be confusing to anybody not using any custom scripts (which would probably be the majority).  I would recommend checking if the count of `scripts/*.sh` is greater than 0 before displaying this `else` message.